pkg/netns/netns_linux.go

```diff
@@ -50,7 +50,9 @@ import (
 
 // newNS creates a new persistent (bind-mounted) network namespace and returns the
 // path to the network namespace.
-func newNS(baseDir string) (nsPath string, err error) {
+// If pid is not 0, returns the netns from that pid persistently mounted. Otherwise,
+// a new netns is created.
+func newNS(baseDir string, pid uint32) (nsPath string, err error) {
```

Comment:
What is the condition where the pid could be 0?

Reply:
Here: https://github.com/containerd/containerd/pull/7679/files#diff-56fc249a3471d2c7a340d76adbfda49d6e66e3efc8b50245762938a996a260c1R181